app-text/ayat_tarajem_data/ayat_tarajem_data-0.0.1.recipe

```diff
@@ -0,0 +1,57 @@
+SUMMARY="Translation data from KSU-Electronic Mosshaf project"
+DESCRIPTION="Ayat data is use by other project like elforkane and mtafsir \
+get from original project to minimize maintenance .
+noted that the file is derivation from project tanzil.net"
+HOMEPAGE="http://quran.ksu.edu.sa/ayat/"
+COPYRIGHT="2014-2019 King Saud University
+           2008-2016 Tanzil.net team
+		   "
+LICENSE="Creative Commons Attribution 3.0"
```

Comment:
The license text is quite different from https://creativecommons.org/licenses/by/3.0/legalcode
I would suggest another name for the license (for instance the recipe name)

Reply:
ok . change the license.. 
thanks @korli